recipes/ipypublish/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "ipypublish" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e1605c8462e8eb1e5066df7f55abc6e69b7694fd802e30273b3aa8976214887
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - nbpublish = ipypublish.frontend.nbpublish:run
+    - nbpresent = ipypublish.frontend.nbpresent:run
+    - ipubpandoc = ipypublish.filters_pandoc.main:pandoc_filters
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - bibtexparser
+    - docutils
+    - jinja2
+    - jsonextended >=0.7
+    - jsonschema
+    - jupytext
+    - nbconvert
+    - nbformat
+    - pandoc ==2.6
```

Comment:
Does pandoc need to be exactly version 2.6?

Reply:
This is actually something I wanted to ask about:
No, there is no particular version requirement on pandoc,
although 2.6 is what all the tests are run on, so I guess it is recommended.

However, the reason I set a version, is that the [nbconvert feedstock](https://github.com/conda-forge/nbconvert-feedstock/blob/master/recipe/meta.yaml) sets a restriction on pandoc<2.0.
There does not appear any reason why this should be the case 
(and an issue has been raised jupyter/nbconvert#928), and essentially I wanted to 'override' it,
such that you don't end up with an unnecessarily old version of pandoc.

I was actually surprised that this worked:

> conda create -n test nbconvert "pandoc>=2.6"

It seems that conda does not strictly enforce version pinning from dependencies?

Reply:
> It seems that conda does not strictly enforce version pinning from dependencies?

The version constraint on `pandoc` [was added to `nbconvert` in `5.4.0`](https://github.com/conda-forge/nbconvert-feedstock/issues/22). This causes `conda` to look for a compatible version with `pandoc >=2.6` and results in an older version of `nbconvert` being installed (`5.3.1`).

If it isn't a problem, I would suggest removing the version requirement here and pushing for it to be fixed "properly" upstream in jupyter or at least in the nbconvert feedstock.

If it is a problem, pin the version to a range like `pandoc >=2.2`.

Reply:
Ah, didn't notice the change in version, that makes more sense. I have removed the version pinning

Reply:
Yeah it can be annoying occasionally ancient versions are resurrected as they weren't pinned correctly. (There is a mechanism to fix for when it's a major issue.)

LGTM 👍